### ISSUE TITLE
Consider enhancing plugin to warn when using bean-construct with multiple method matches

### ISSUE THREAD

**Issue by askarby** [camel-tooling/camel-idea-plugin#394]
Consider adding functionality to warn, when bean-endpoint is used with ambiguous matches... given the following example:

Bean:
```java
class SomeBean {
    public void awesomeFunctionality(String name) {
         ....
    }
    public void awesomeFunctionality(String name, String organization) {
         ....
    }
}
```

Route (Snippet):
```java
...
.bean(SomeBean.class, "awesomeFunctionality")
...
```

**Comment by fharms** [comment]
How should this work, like an icon in the gutter ?

**Comment by fharms** [comment]
Fixed in #427 This will mark the bean method with an red line, just like you will see with normal compile errors, but it will not prevent the compiler or execute the code.
